cli/src/cmd/forge/cache.rs

```diff
@@ -41,25 +42,9 @@ pub struct CleanArgs {
     #[clap(
         env = "CHAIN",
         default_value = "all",
-        possible_values = [
-            "all",
-            "mainnet",
-            "ropsten",
-            "rinkeby",
-            "goerli",
-            "kovan",
-            "xdai",
-            "polygon",
-            "polygon_mumbai",
-            "avalanche",
-            "avalanche_fuji",
-            "sepolia",
-            "moonbeam",
-            "moonbeam_dev",
-            "moonriver",
-            "optimism",
-            "optimism-kovan"
-    ])]
+        possible_value = "all",
+        possible_values = Chain::VARIANTS
```

Comment:
Hmm, AFAIK we already have a `ClapChain` enum somewhere that does this without strum - we should just use that. Also, we should do that everywhere we accept a chain name.

Reply:
oh whoops, will take a look at using this